packages/terra-form-select/src/_Frame.scss

```diff
@@ -0,0 +1,252 @@
+@import '~terra-mixins';
+
+:local {
+  .select {
+    background-color: var(--terra-select-background-color, #fff);
+    color: var(--terra-select-color, #1c1f21);
+    display: flex;
+    font-size: var(--terra-select-font-size, 1.143rem);
+    line-height: var(--terra-select-line-height, 1.75);
+  }
+
+  .default {
+    background-size: var(--terra-select-background-size, 0 0);
+    border: var(--terra-select-border, 1px solid #dedfe0);
+    border-radius: var(--terra-select-border-radius, 3px);
+    cursor: pointer;
+    padding: var(--terra-select-padding, 0 0 0 0.42857rem);
```

Comment:
Let's split this out into individual styles (padding-top, padding-right, padding-left, padding-bottom). The RTL postcss plugin doesn't seem to process this as a bidi based style as it is now.